apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Supabase Logs can now be self-hosted
```

Comment:
- Supabase Logs is now open source
- Self-hosted Logging Server 
- Supabase Logs: an open source logging server
- Supabase Logs: an open source Sentry alternative
- supabase logging is now integrated
- Supabase logs is now self-hostable
- Supabase Logs is integrated into the Supabase Stack

Reply:
Let me know which one 😁